pkg/config/execution_unit.go

```diff
@@ -12,14 +12,13 @@ type (
 		Type                 string            `json:"type" yaml:"type" toml:"type"`
 		NetworkPlacement     string            `json:"network_placement,omitempty" yaml:"network_placement,omitempty" toml:"network_placement,omitempty"`
 		EnvironmentVariables map[string]string `json:"environment_variables,omitempty" yaml:"environment_variables,omitempty" toml:"environment_variables,omitempty"`
-		HelmChartOptions     *HelmChartOptions `json:"helm_chart_options,omitempty" yaml:"helm_chart_options,omitempty" toml:"helm_chart_options,omitempty"`
+		HelmChartOptions     HelmChartOptions  `json:"helm_chart_options,omitempty" yaml:"helm_chart_options,omitempty" toml:"helm_chart_options,omitempty"`
```

Comment:
By removing the pointer, if we modify this somehwere wouldnt it not persist? like we set values here when we do analysis on certain things which we will need to stay in here

Reply:
You can still modify it: https://go.dev/play/p/yon-_o93FT8